website/docs/d/privatelink_endpoint_service_data_federation_online_archive.html.markdown

```diff
@@ -46,6 +46,8 @@ In addition to all arguments above, the following attributes are exported:
 * `type` - Human-readable label that identifies the resource type associated with this private endpoint.
 * `comment` - Human-readable string to associate with this private endpoint.
 * `provider_name` - Human-readable label that identifies the cloud service provider. 
+* `region` - Physical location of your MongoDB cluster. The region you choose can affect network latency for clients accessing your databases.  Requires the **Atlas region name**, see the reference list for [AWS](https://docs.atlas.mongodb.com/reference/amazon-aws/), [GCP](https://docs.atlas.mongodb.com/reference/google-gcp/), [Azure](https://docs.atlas.mongodb.com/reference/microsoft-azure/).
```

Comment:
@EspenAlbert once you are aligned can you make sure to update the 2 Data Sources files to align definitions edits made in Resource file for paramters `customer_endpoint_dns_name` and `endpoint_id`